selftests/functional/test_plugin_jobscripts.py

```diff
@@ -93,14 +106,37 @@ def test_status_non_zero(self):
         config = script.TemporaryScript("non_zero.conf",
                                         SCRIPT_NON_ZERO_CFG % self.pre_dir)
         with config:
-            cmd = './scripts/avocado --config %s run passtest.py' % config
+            cmd = './scripts/avocado --config %s run /bin/true' % config
```

Comment:
BTW. Currently last test failed due to extra complain from log handler
$python selftests/functional/test_plugin_jobscripts.py
...F
======================================================================
FAIL: test_status_non_zero (__main__.JobScriptsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "selftests/functional/test_plugin_jobscripts.py", line 115, in test_status_non_zero
    result.stderr)
AssertionError: 'Pre job script "/tmp/avocado___main__ceUeTa/pre.d/non_zero.sh" exited with status "1"\n' != 'Pre job script "/tmp/avocado___main__ceUeTa/pre.d/non_zero.sh" exited with status "1"\nNo handlers could be found for logger "avocado.test.stderr"\n'